src/dlgPackageExporter.cpp

```diff
@@ -279,7 +279,7 @@ void dlgPackageExporter::slot_export_package()
             }
             QString fullName = tempDir + "/" + contents[i];
             struct zip_source* s = zip_source_file(archive, fullName.toStdString().c_str(), 0, 0);
-            if (s == NULL) {
+            if (s == nullptr) {
```

Comment:
Technically, like the `pcre` library the `zip` one is also `C` code so NULL might indeed be what it returns...